src/components/Navigation/Ui/EditableFieldRenderer.js

```diff
@@ -36,37 +34,34 @@ class EditableFieldRenderer extends Component {
   }
 
   componentDidMount() {
-    const { definition, previewDefinition, curationSuggestions, field, type } = this.props
-    this.setComputedValue(definition, previewDefinition, curationSuggestions, field, type)
+    const { definition, previewDefinition, field, type } = this.props
+    this.setComputedValue(definition, previewDefinition, field, type)
```

Comment:
In this file i've removed some old implementations of the first attempt for the Suggestions behavior